module/VuFind/tests/unit-tests/src/VuFindTest/Search/Blender/ParamsTest.php

```diff
@@ -1069,9 +1069,13 @@ protected function getConfigManager()
      */
     protected function getParamsClassesArray(): array
     {
-        $solrConfigMgr = $this->createMock(\VuFind\Config\PluginManager::class);
+        $solrConfigMgr = new \VuFindTest\Container\MockConfigPluginManager($this);
+        $solrConfigMgr->set('facets', new \Laminas\Config\Config([]));
+        $solrConfigMgr->set(
+            'searches',
+            new \Laminas\Config\Config(['General' => ['tie_breaker_sort' => 'id']])
```

Comment:
What happens if we remove these customizations to the configuration of the configuration plugin manager? Does the test actually fail, or did you just add these in order to test the new functionality? If the former, I'm curious how/why; if the latter, this might not be the best place for this test, since this is a test of Blender, and it would probably make more sense to put a test in the Solr-specific test class.

Another somewhat related question is whether perhaps we want the tie-breaker field to default to "id" if not otherwise configured. In that case, I would expect that we would need to change this test, but not customize the search configuration in the test.